cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-slack-webhook-notification.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: appstudio.redhat.com/v1beta1
```

Comment:
not needed in this workspace. the pipeline we are targeting does not support the slack task atm. We could add this later.

Reply:
Agreed. Dropped in 094b658.